dropwizard-views/src/main/java/io/dropwizard/views/ViewMessageBodyWriter.java

```diff
@@ -82,12 +76,9 @@ public void writeTo(View t,
                 }
             }
             throw new ViewRenderException("Unable to find a renderer for " + t.getTemplateName());
-        } catch (Exception e) {
+        } catch (ViewRenderException e) {
             LOGGER.error("Template Error", e);
```

Comment:
That makes sense. I've removed the logging from `ViewMessageBodyWriter` and added the logging code to `ViewRenderExceptionMapper` to preserve the original exception logging behavior.